demisto_sdk/commands/validate/initializer.py

```diff
@@ -578,13 +585,20 @@ def get_items_status(
                     statuses_dict[path] = git_status
             elif PACKS_PACK_META_FILE_NAME in path_str:
                 statuses_dict[path] = git_status
-            elif self.is_pack_item(path_str):
-                metadata_path = self.obtain_metadata_path(path)
-                if metadata_path not in statuses_dict:
-                    statuses_dict[metadata_path] = None
-            else:
+            elif not self.is_pack_item(path_str):
```

Comment:
Since we have several nested if/elif here I would appreciate if you could add a comment explaining each of the, for clarity